src/account/GoldEducation.tsx

```diff
@@ -51,7 +50,6 @@ export default function GoldEducation() {
 
 GoldEducation.navigationOptions = {
   ...noHeader,
-  ...TransitionPresets.ModalTransition,
```

Comment:
I think all screens that were displayed with a modal transition should be moved to a modal stack. But we can do this separately.